usr/share/rear/conf/default.conf

```diff
@@ -3377,7 +3377,10 @@ ELILO_BIN=
 ################ ---- custom scripts
 #
 # NOTE: The scripts can be defined as an array to better handly spaces in parameters.
```

Comment:
If we decide to merge your PR as it is, the line above should be deleted, because it will not correspond to the reasoning for using an array anymore (and has never been really correct anyway).